tests/integration/test_github/test_configuration/data/test_cases/cases_invalid_configuration.yaml

```diff
@@ -0,0 +1,120 @@
+    TIME_DELAY: '1s'
+    ORG_NAME: 'dummy'
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'enabled'
+    error_type: 'Invalid'
+
+- name: Invalid_Only_Future_Events
+  description: Check github integration does not start with Invalid Only Future Events
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'invalid'
+    INTERVAL: '1m'
+    CURL_MAX_SIZE: '1024'
+    TIME_DELAY: '1s'
+    ORG_NAME: 'dummy'
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'only_future_events'
+    error_type: 'Invalid'
+
+- name: Invalid_Interval
+  description: Check github integration does not start with Invalid Interval
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'yes'
+    INTERVAL: '-5s'
+    CURL_MAX_SIZE: '1024'
+    TIME_DELAY: '1s'
+    ORG_NAME: 'dummy'
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'interval'
+    error_type: 'Invalid'
+    
+- name: Invalid_Curl_Max_Size
+  description: Check github integration does not start with Invalid Curl Max Size
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'yes'
+    INTERVAL: '1m'
+    CURL_MAX_SIZE: '-12'
+    TIME_DELAY: '1s'
+    ORG_NAME: 'dummy'
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'curl_max_size'
+    error_type: 'Invalid'
+
+- name: Invalid_Time_Delay
+  description: Check github integration does not start with Invalid Time Delay
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'yes'
+    INTERVAL: '1m'
+    CURL_MAX_SIZE: '1024'
+    TIME_DELAY: '7k'
+    ORG_NAME: 'dummy'
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'time_delay'
+    error_type: 'Invalid'
+
+- name: Empty_Org_Name
+  description: Check github integration does not start with Invalid Org Name
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'yes'
+    INTERVAL: '1m'
+    CURL_MAX_SIZE: '1024'
+    TIME_DELAY: '1s'
+    ORG_NAME: ''
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'org_name'
+    error_type: 'Empty'
+
+- name: Empty_Api_Token
+  description: Check github integration does not start with Invalid Api Token
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'yes'
+    INTERVAL: '1m'
+    CURL_MAX_SIZE: '1024'
+    TIME_DELAY: '1s'
+    ORG_NAME: 'dummy'
+    API_TOKEN: ''
+    EVENT_TYPE: 'all'
+  metadata:
+    event_monitor: 'api_token'
+    error_type: 'Empty'
+
+- name: Invalid_Event_Type
+  description: Check github integration does not start with Invalid Event Type
+  configuration_parameters:
+    ENABLED: 'yes'
+    ONLY_FUTURE_EVENTS: 'yes'
+    INTERVAL: '1m'
+    CURL_MAX_SIZE: '1024'
+    TIME_DELAY: '1s'
+    ORG_NAME: 'dummy'
+    API_TOKEN: 'token'
+    EVENT_TYPE: 'invalid'
+  metadata:
+    event_monitor: 'event_type'
+    error_type: 'Invalid'
+    
```

Comment:
Does it have a new line at the end?

Reply:
Fixed with this [commit](https://github.com/wazuh/wazuh/pull/17621/commits/744e21ccdbe9566a401879d6ae6cbbc3edcb7adb)